ansible/roles/common/tasks/setup_runtime_directories.yml

```diff
@@ -30,7 +30,12 @@
     owner={{ application_user }}
     group={{ application_group }}
 
-- name: Copy application configuration
+- name: Copy application configuration (docker)
   template: src=etc/pouta_blueprints/app_config.j2
             dest={{ application_config }}
+  when: deploy_mode == "docker"
 
+- name: Copy application configuration (devel)
```

Comment:
I think we are better off using one config template, the differences are quite easily handled by vars and templates and aliases in /etc/hosts in devel deployment